backend/src/main/resources/templates/RegistrationConfirmation.ftl

```diff
@@ -327,7 +327,7 @@
                                     </p>
                                     <p>
 
-                                        Tomáš Kouřim
+                                        Ing. Tomáš Kouřim, CEO Mild Blue
```

Comment:
Mozna sem dame 
`MUDr. Alžběta Honzková, Mild Blue`
To bude vypadat erudovane

Reply:
jsem pro, to vypada cool

a pak teda bych ten text asi zarovnal do bloku, aby to nevypadalo takhle rozpadly